index.html

```diff
@@ -219,6 +219,91 @@ <h3>Skills</h3>
 				</div>
 			</div>
 		</section>
+		<section id="contact">
+			<div class="pri-text">
+				<p class="contact-para">
+					I'm always interested in hearing about new projects, so if you'd like
+					to chat please get in touch.
+				</p>
+			</div>
+			<form action="https://formspree.io/f/xpznwejk" method="post" class="form">
+				<fieldset>
+					<label class="firstN"
+						><input
+							type="text"
+							name="firstN"
+							id="firstN"
+							placeholder="First name"
+							required
+						/>
+					</label>
+					<label class="lastN">
+						<input
+							type="text"
+							name="lastN"
+							id="lastN"
+							placeholder="Last Name"
+							required
+						/>
+					</label>
+					<label class="fullN"
+						><input
+							type="text"
+							name="fullname"
+							id="fullname"
+							placeholder="Full name"
+							required
+						/>
+					</label>
+
+					<label class="email"
+						><input
+							type="email"
+							name="email"
+							id="email"
+							placeholder="Email address"
+							required
+						/>
+					</label>
+					<label class="msg">
+						<textarea
+							name="msg"
+							id="msg"
+							cols="30"
+							rows="10"
+							maxlength="500"
+							minlength="30"
+							placeholder="Enter text here"
+							required
+						></textarea>
+					</label>
+					<button class="btn" type="submit">Get in Touch</button>
+				</fieldset>
```

Comment:
- As stated by the previous reviewer the `background color ` you gave your button is different from the one we have in the Figma design, kindly check the color type we have in the Figma design and apply it in your project.

Reply:
Hello @brainconnect93, can you please double check because i used the exact background color #36B37F; the form submits when i submit and also the image link is working... As requested by the previous reviewer, i have made all the changes and they are working perfectly on my side

Reply:
@brainconnect93 I would be happy to demonstrate all the changes I have made, please take a look at the code again...and also in the first comment you mentioned @zeff96, which is not me, so i think there must be a mix up in your review...because the code i have submitted in my review is working perfectly and i don't think the changes mentioned above really apply